### PR TITLE
fix(cli): error message for non-existent tf cloud orgs

### LINKED ISSUE #2982: Error: [select prompt] No selectable choices. All choices are disabled.
### Expected Behavior

While running cdktf init --template="typescript" a terraform cloud project directory should have been initialized.

### Actual Behavior

Error: [select prompt] No selectable choices. All choices are disabled.
    at C:\Users\redacted\AppData\Roaming\npm\node_modules\cdktf-cli\bundle\bin\cmds\handlers.js:460:259
    at Ki (C:\Users\redacted\AppData\Roaming\npm\node_modules\cdktf-cli\bundle\bin\cmds\handlers.js:454:522)
    at C:\Users\redacted\AppData\Roaming\npm\node_modules\cdktf-cli\bundle\bin\cmds\handlers.js:460:216
    at C (C:\Users\redacted\AppData\Roaming\npm\node_modules\cdktf-cli\bundle\bin\cmds\handlers.js:454:1793)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

### Steps to Reproduce

1. npm install -g typescript
2. npm install -g cdktf-cli
3. cdktf init --template="typescript"

### Versions

language: null
cdktf-cli: 0.17.0
node: v18.16.1
terraform: 1.5.1
arch: x64
os: win32 10.0.22621

### Providers

_No response_

### Gist

_No response_

### Possible Solutions

No

### Workarounds

No

### Anything Else?

No

### References

None

### Help Wanted

- [ ] I'm interested in contributing a fix myself

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### PR DESCRIPTION
**Pull request by howdoicomputer**
### Related issue

Fixes #2982

### Description

If a user is not actively part of an organization in Terraform Cloud then the CLI will error out with a stacktrace that obfuscates the problem. This bug fix returns a happy error message instead.

A user not being part of an organization may be common if they're going through tutorials.

### Checklist

- [x] I have updated the PR title to match [CDKTF's style guide](https://github.com/hashicorp/terraform-cdk/blob/main/CONTRIBUTING.md#pull-requests-1)
- [x] I have run the linter on my code locally
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation if applicable
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works if applicable
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: examples/java/aws/imports/constraints.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (howdoicomputer) [inline examples/java/aws/imports/constraints.json:1]: These were generated by git hooks. I'm not sure if that's intentional. I can remove them if not.

File: packages/cdktf-cli/src/bin/cmds/helper/init.ts [hunk 2]
```diff
@@ -372,6 +372,12 @@ async function gatherInfo(
       token
     );
 
+    if (organizationIds.length == 0) {
+      throw Errors.Usage(
+        `You must be part of an organization in Terraform Cloud in order to use it as a RemoteBackend.\n`
```
Comment (DanielMSchmidt) [inline packages/cdktf-cli/src/bin/cmds/helper/init.ts:377]:
```suggestion
        `You must be part of an organization in Terraform Cloud in order to use it as a RemoteBackend or CloudBackend.\n`
```

**Review by DanielMSchmidt — COMMENTED** [review]
Thanks a ton! That error message will be so much better!

**Comment by howdoicomputer** [comment]
Happy to help!

**Comment by DanielMSchmidt** [comment]
Hey, thanks for your help! I moved this into a different PR so that I could run prettier :)

**Pull request closed without merging**